src/xnatdcat/cli_app.py

```diff
@@ -90,6 +90,18 @@ def __parse_cli_args():
         help="Path of logfile to use. Default is xnatdcat.log in current directory",
     )
 
+    # Both opt-in and opt-out at the same time is not very logical, so it is not allowed.
+    group = parser.add_mutually_exclusive_group()
```

Comment:
don't you missing required=True here so at least one is provided or these arguments are optional? what will happen if both are none?

Reply:
Ah, OK, it seemed from the PR description that it is for every instance.